framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -4424,6 +4452,65 @@ void VulkanReplayConsumerBase::OverrideGetImageSubresourceLayout(
                                          image_info->allocator_data);
 }
 
+VkResult VulkanReplayConsumerBase::OverrideCreateRenderPass(
+    PFN_vkCreateRenderPass                                      func,
+    VkResult                                                    original_result,
+    const DeviceInfo*                                           device_info,
+    const StructPointerDecoder<Decoded_VkRenderPassCreateInfo>* pCreateInfo,
+    const StructPointerDecoder<Decoded_VkAllocationCallbacks>*  pAllocator,
+    HandlePointerDecoder<VkRenderPass>*                         pRenderPass)
+{
+    GFXRECON_UNREFERENCED_PARAMETER(original_result);
+
+    return swapchain_->CreateRenderPass(func,
+                                        device_info,
+                                        pCreateInfo->GetPointer(),
+                                        GetAllocationCallbacks(pAllocator),
+                                        pRenderPass->GetHandlePointer());
+}
+
+VkResult VulkanReplayConsumerBase::OverrideCreateRenderPass2(
+    PFN_vkCreateRenderPass2                                      func,
+    VkResult                                                     original_result,
+    const DeviceInfo*                                            device_info,
+    const StructPointerDecoder<Decoded_VkRenderPassCreateInfo2>* pCreateInfo,
+    const StructPointerDecoder<Decoded_VkAllocationCallbacks>*   pAllocator,
+    HandlePointerDecoder<VkRenderPass>*                          pRenderPass)
+{
+    GFXRECON_UNREFERENCED_PARAMETER(original_result);
+
+    return func(device_info->handle,
```

Comment:
Does this need to call `swapchain_->CreateRenderPass2`?

Reply:
fixed